src/CMakeLists.txt

```diff
@@ -706,7 +706,7 @@ list( APPEND SOURCES
       >
 
       $<$<BOOL:${USE_LIBMAD}>:
-         import/ImportMP3.cpp
+         import/ImportMP3_MAD.cpp
```

Comment:
The old file name is still mentioned in comments in itself and in Import.cpp.

Reply:
I will check!

Reply:
Anyway, I have restructured commits to improve it